help/md/steppingStoneSampler.md

```diff
@@ -1,9 +1,93 @@
 ## name
 steppingStoneSampler
 ## title
+Stepping-stone marginal likelihood estimation
 ## description
+Applies the stepping-stone technique (Fan et al. 2011; Xie et al. 2011) to
+estimate the marginal likelihood of a model from a series of unnormalized
+"power posterior" densities in which the likelihood term is raised to a power
+B between 0 and 1.
 ## details
+For K power posteriors starting with the posterior proper (B_1 = 1) and ending
+with the prior (B_K = 0), the stepping-stone marginal likelihood estimate r_SS
+is equal to:
+
+    r_SS = \prod_{k=1}^K r_{SS,k}
+    
+where
+
+    r_{SS,k} = (1/n) * L_{max,k}^(B_{k-1} - B_k) * \sum_{i=1}^n (L_{i,k - 1} / L_{max,k})^(B_{k-1} - B_k)
+    
+where n denotes the number of MCMC samples from each power posterior,
+L_{max,k} denotes the highest likelihood sampled for the k-th power posterior,
+and L_{k,i} denotes the likelihood of the i-th MCMC sample from the k-th power
+posterior (Xie et al. 2011: 153). If we instead assume that L_{max,k} and
+L_{i,k} are given on the log scale, and that r_SS is to be expressed on the
+log scale as well, this yields:
+
+    r_SS = \sum_{k=1}^K L_{max,k} * (B_k - B_{k-1}) + log( \sum_{i=1}^n exp( (L_{i,k-1} - L_{max,k}) * (B_{k-1} - B_k) ) / n )
+    
+which is how the method is implemented in RevBayes. The estimate is returned by
+calling the `.marginal()` method on the sampler object. We can also compute the
+standard error of this estimate in two different ways using the `.stdError()`
+method. By default, the method uses the delta approximation suggested by Xie et 
+al. (2011: 153):
+
+    SE = sqrt( \sum_{k=1}^K 1/r_{SS,k}^2 * Var( r_{SS,k} )/N_{eff,k} )
+    
+where N_{eff,k} is the effective sample size for the k-th power posterior. The
+approximation may be unreliable if any individual summand exceeds ~0.1, which
```

Comment:
It's not clear that there's an easy way (aside from manually inspecting individual files) for a user to determine when this is the case.  Would it be possible to output a warning when this case is triggered, to the effect that "standard error approximation may be unreliable; try `bootstrap = TRUE`"?

Reply:
We actually already do this: the warning looks as follows:

```
> ss.stdError()
   Warning: Standard error approximation unreliable as var(r_k)/r_k^2 > 0.1 for the following powers:
   0.188881, 0.112215, 0.071827, 0.0627474, 0.0585658, 0.0546101, 0.0508711, 0.0473398.
   
   1.741568
```

That said, I agree it's a great idea for the warning message to recommend using `bootstrap = TRUE` instead!